docs/source/data/data_catalog.md

```diff
@@ -337,7 +337,7 @@ The list of all available parameters is given in the [Paramiko documentation](ht
 
 ## Creating a Data Catalog YAML configuration file via CLI
 
-You can use [`kedro catalog create` command](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file) to create a Data Catalog YAML configuration.
+You can use [`kedro catalog create` command to create a Data Catalog YAML configuration](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file).
 
 It creates a `<conf_root>/<env>/catalog/<pipeline_name>.yml` configuration file with `MemoryDataSet` datasets for each dataset in a registered pipeline if it is missing from the `DataCatalog`.
```

Comment:
This creates a `<conf_root>/<env>/catalog/<pipeline_name>.yml` configuration file